src/apiCalls.js

```diff
@@ -6,7 +6,7 @@ export function postHydrationData(userId, date, amount){
   let body = {
     "userID": userId, 
     "date": date, 
-    "numOunces": amount
+    "numOunces": parseFloat(amount)
```

Comment:
nice fix! Since it was going through as a string, the plus sign in the function was actually concatenating the string to the avg number, then type coercing it to a much larger number than it should have been.